.github/workflows/screeners.yml

```diff
@@ -0,0 +1,217 @@
+name: Screeners workflow
+
+on: pull_request
+
+env:
+        SKIPCOMPONENTGOVERNANCEDETECTION: true
+
+        ConnectionString: ${{ secrets.AZURE_STORAGE_CONNECTION_STRING}}
```

Comment:
```suggestion
```

This is looking weird, can we try the two things again (now that we have ironed out  errors:

* call the env variable `AZURE_STORAGE_CONNECTION_STRING` and remove `--connection-string` argument
* or remove the `--connection-string` argument completely because according the CLI docs it should work 

![image](https://user-images.githubusercontent.com/20744592/180398325-b5bbb531-ab6b-4367-beed-d8b151111899.png)